newclient/src/main.ts

```diff
@@ -45,7 +46,21 @@ const wsLink = new WebSocketLink(subscriptionClient)
 const onErrorLink = onError(({ graphQLErrors }) => {
     // Log every GraphQL errors in develop
     if (graphQLErrors && process.env.NODE_ENV !== 'production') {
-        graphQLErrors.map(({ message, locations, path }) => console.log(`[GraphQL error]: Message: ${message}, Location: ${locations}, Path: ${path}`))
+        graphQLErrors.map(({ message, locations, path }) => {
+            const newError = `[GraphQL error]: Message: ${message}, Location: ${locations}, Path: ${path}`
+            console.log(newError)
+        })
+    }
+    if (graphQLErrors && process.env.NODE_ENV === 'production') {
+        graphQLErrors.map(({ message, locations, path }) => {
+            const newError = `[GraphQL error]: Message: ${message}, Location: ${locations}, Path: ${path}`
```

Comment:
you should do 
```
const newError = new Error(`[GraphQL error]: Message: ${message}, Location: ${locations}, Path: ${path}`)
```

then check whether `newError.message` contains blockNotMined...etc

Reply:
will change !